platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -45,9 +62,30 @@ NS_ASSUME_NONNULL_BEGIN
  as possible to avoid affecting performance.
 
  @param mapView The map view whose viewpoint is changing.
+
+ @note If `mapView:regionIsChangingWithReason:` is implemented this method will not be called.
```

Comment:
We generally refrain from using `@note` except for very important notes. This particular precondition is a bit of a trap, so `@note` might be warranted. However, we go with my suggestion above to call both variants in succession and document the order, then that information can go in an ordinary paragraph above the `@param` commands.